.github/workflows/report_quiz/main.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+set -e
+
+BASE_DIR=$(dirname "${BASH_SOURCE[0]}")
+DATABASE_DIR="${BASE_DIR}/database"
+ARCHIVE_DIR="${BASE_DIR}/archive"
+TEMPFILE="$(mktemp)"
+
+# load functions
+source "${BASE_DIR}/common/utils.sh"
+source "${BASE_DIR}/quiz/functions.sh"
+source "${BASE_DIR}/leaderboard/functions.sh"
+
+# load environement variables
+source "${BASE_DIR}/env.sh"
```

Comment:
Add a -rf check for this file
And exit 1 if it's not present before continuing

Reply:
Fail fast is enabled at the top of the `main.sh`

Reply:
I prefer handling personally exeptions and explicit errors messages to have full control when am building blocks in a codebase, but okay... Let's keep it like that.

To go deeper, the source will fails with something like "env.sh No such file or directory" (I guess)... But in an another composes scenario, that error is not going to reflect a user error meaningful enought to fix without relooking what the script does